### PR TITLE
fixed histogram's url

### PR DESCRIPTION
**Pull request by mcarmonaa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -157,7 +157,7 @@ Enry's language detection has been compared with Linguist's language detection.
 
  Following results were obtained:
 
-![histogram](https://raw.githubusercontent.com/src-c/enry/master/benchmarks/histogram/distribution.jpg)
+![histogram](https://raw.githubusercontent.com/src-c/enry/master/benchmarks/histogram/distribution.png)
```
Comment (abeaumont) [inline README.md]: This url is not working for me either. Why not just [histogram](benchmarks/histogram/distribution.png)?

Comment (mcarmonaa) [inline README.md]: @abeaumont  The `!` is to embed images, It is working for me, how are you testing it?

Comment (abeaumont) [inline README.md]: @mcarmonaa https://raw.githubusercontent.com/src-c/enry/master/benchmarks/histogram/distribution.png

Comment (abeaumont) [inline README.md]: If it worked, it should be viewable at https://github.com/mcarmonaa/enry/tree/fix/histogram-url, shouldn't it?

Comment (mcarmonaa) [inline README.md]: Oops, s/src-c/src-d/

**Pull request merged**